From c62f9306347f6d27a9130547693e1e456ac64a72 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakub=20=C4=8Cajka?= <jcajka@redhat.com>
Date: Fri, 1 Sep 2023 09:31:39 +0200
Subject: [PATCH] osasnmpd: Fix missing semicolon
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

5.9.4 net-snmp started to require semicolon on the config_require there
are no docs covering this change.

Link: https://bugzilla.redhat.com/show_bug.cgi?id=2235734
Closes: https://github.com/ibm-s390-linux/s390-tools/pull/155
Signed-off-by: Jakub Čajka <jcajka@redhat.com>
Signed-off-by: Jan Höppner <hoeppner@linux.ibm.com>
---
 osasnmpd/ibmOSAMib.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/osasnmpd/ibmOSAMib.h b/osasnmpd/ibmOSAMib.h
index 6dc2345d5..2b20684b0 100644
--- a/osasnmpd/ibmOSAMib.h
+++ b/osasnmpd/ibmOSAMib.h
@@ -16,7 +16,7 @@
 
 /* we may use header_generic and header_simple_table from the util_funcs module */
 
-config_require(util_funcs)
+config_require(util_funcs);
 
 
 /* function prototypes */
